cmd/podman/utils/signals_linux.go

```diff
@@ -0,0 +1,14 @@
+// +build !windows
+
+package utils
+
+import (
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+// Platform specific signal synonyms
+var (
+	Hup os.Signal = unix.SIGHUP
```

Comment:
```suggestion
	SIGHUP os.Signal = unix.SIGHUP
```

Might be easier to understand when the identifier is consistent.